hooks/use-bot-stats.ts

```
@@ -0,0 +1,402 @@
+      // Use the real API instead of mock data
+      return fetchBotStats(queryParams)
+    },
+    staleTime: 1000 * 60 * 5,
+  })
+
+  const data = useMemo(() => {
+    if (!apiData?.bots) return null
+
+    const botsWithPlatform: FormattedBotData[] = apiData.bots.map((bot) => ({
+        ...bot,
+      platform: getPlatformFromUrl(bot.meeting_url) as any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)